dist/newrelic/newrelic.yml

```diff
@@ -0,0 +1,302 @@
+# This file configures the New Relic Agent.  New Relic monitors
```

Comment:
My concerns are twofold:

1. We spent a lot of time configuring New Relic. The new configs should match the existing ones. If we're throwing away the existing configs, we should have a good reason for it.

2. If we need to deploy to Bridge every time we want to tweak the New Relic monitoring, this will severely slow down our ability to change our monitoring settings. If the actual configured monitoring settings are always going to be out-of-sync with source control, why have them in source control in the first place?